src/manage_sql.c

```diff
@@ -30081,6 +30133,18 @@ print_report_xml_start (report_t report, report_t delta, task_t task,
            (strchr (levels, 'f') ? orig_f_false_positives : 0));
   else
     {
+      if (count_filtered)
+        {
+          debugs = f_debugs;
+          holes = f_holes;
+          infos = f_infos;
+          logs = f_logs;
+          warnings = f_warnings;
+          false_positives = f_false_positives;
+          filtered_result_count = debugs + holes + infos + logs + warnings
+                                  + false_positives;
```

Comment:
Why are you overwriting the full counts with the filtered ones here instead of just adding the filtered counts for `filtered_result_count`?
Earlier you already get them with `report_counts_id_full (...)` and while the total is still correct after this the counts per level are not if there are any filter conditions limiting the results selection.

Reply:
Oh I misunderstood what the report_counts_id part was doing.  Should be right with 
299fefa.  Thanks for catching.  Really want to clean this code up, but need to backport these PRs to gvmd 8.